hazelcast-client-new/src/test/java/com/hazelcast/client/ClientConnectionTest.java

```diff
@@ -42,36 +41,37 @@
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(HazelcastSerialClassRunner.class)
+@RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientConnectionTest extends HazelcastTestSupport {
 
+    private final TestHazelcastFactory hazelcastFactory = new TestHazelcastFactory();
```

Comment:
Perhaps the HazelcastTestSupport should be cleaned up.

The main idea is that for a test you want to impose the least amount of environmental setup. Having an additional factory field violates this idea. Otherwise people will be copy pasting this line for every test they make and isn't very appealing. 

Apart from that I would like to see the setup of non-clients tests to be the same as client tests.